src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */ 
+import Big from 'big.js';
```

Comment:
- [ ] Kindly notice that you should add `big.js` in your dependencies as it is asked in the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_events.md#:~:text=note%3A%20you%20need%20to%20add%20big.js%20as%20dependency), as it is shown error because of that.

![Capture](https://user-images.githubusercontent.com/33752735/199546603-a08619c9-2397-4519-ac39-b1814366d88b.PNG)